playgame.py

```diff
@@ -180,7 +190,7 @@ def play_disclose(self, player_move):
         elif disclose_type == 'rank':
             return self.table.disclose_rank(self.current_player, player_move['player'], player_move['rank'])
 
-    def disqualify(self, bot_move):
```

Comment:
I'm really excited this is the last instance of `bot` in the project. Now ai are called players through the whole thing (according to VSCode search)